styleguide/source/assets/scss/03-organisms/_masthead.scss

```diff
@@ -49,6 +49,18 @@
       align-items: center;
     }
 
+    .ama__h1 {
+      @include gutter($margin-bottom-half...);
+      max-width: 75ch;
```

Comment:
While I applaud your effort to actually enforce character counts, we're actually not allowed to enforce character counts. They're just in the documentation as a "we can't guarantee it will look good past this point but if you want to break the design go ahead". Therefore I'm going to need you to remove it so that they can break the design if they want.